Description: Generate libcomps.pc through libcomps.pc.in.
Author: Mihai Moldovan <ionic@ionic.de>

This is normally supposed to be done by the release team (I guess) through the
build_prep python script. We cannot execute it easily since it requires a full
git repository copy.

Add the processing to the build script logic instead.

--- a/libcomps/src/CMakeLists.txt
+++ b/libcomps/src/CMakeLists.txt
@@ -59,6 +59,8 @@ set (LIB_INST_DIR ${CMAKE_INSTALL_LIBDIR
 ENDIF(NOT LIB_INST_DIR)
 
 
+configure_file("${PROJECT_SOURCE_DIR}/../libcomps.pc.in" "${CMAKE_CURRENT_BINARY_DIR}/libcomps.pc" @ONLY)
+
 install (FILES ${libcomps_HEADERS} DESTINATION include/libcomps)
 install (TARGETS libcomps LIBRARY DESTINATION ${LIB_INST_DIR})
 
